components/eam/chem_proc/inputs/pp_chemUCI_linozv3_mam4_resus_mom_vbs.in

```diff
@@ -88,7 +89,7 @@ SPECIES
  End Solution
 
  Fixed
-   M, N2, O2, H2O, H2, CH4
+   M, N2, O2, H2O, H2, CH4, cnst_O3, cnst_NO3, cnst_OH
```

Comment:
```suggestion
   M, N2, O2, H2O, H2, CH4, prsd_O3, prsd_NO3, prsd_OH
```
Constant (cnst) could be misleading and let one think the fields do not change with time. Prescribed (prsd) is more accurate. Please change other places in the mechanism file and code accordingly.

Reply:
@tangq Thank you. All cnst_xx species have been changed to prsd_xx. @keziming This will likely affect MAM5 as well. If you can change it at this stage in your PR from cnst_OH to prsd_OH, it would be great. Given the tight timeline, it is also fine that we do it later when turning on all features.

Reply:
While the change to "prescribed (prsd)" makes sense to me, I believe the original "cnst" stands for "constituent". Hope the "prsd" won't be interpreted as "personalized" :)

Reply:
Or change to another short form for prescribed which is less likely to be misinterpreted.

Reply:
"prsd" is fine with me. No need to change.